src/components-examples/material/form-field/form-field-prefix-suffix/form-field-prefix-suffix-example.html

```diff
@@ -8,9 +8,8 @@
   </mat-form-field>
 
   <mat-form-field appearance="fill">
-    <mat-label>Amount</mat-label>
-    <input matInput type="number" class="example-right-align">
-    <span matPrefix>$&nbsp;</span>
-    <span matSuffix>.00</span>
```

Comment:
My understanding is that the example tried to show how to use text in the prefix/specifically so I'm not sure that we can replace it with an icon. @mmalerba might have more context here.

Reply:
Yes this is supposed to be showing text prefix/suffix, the form-field above already demonstrates icons